.github/path-ci/run-smoke.sh

```
@@ -0,0 +1,98 @@
+
+echo "== log =="
+out="$(run_capture "$FOUNT" log)"
+assert_output_contains log "$LOG_MARKER" "$out"
+
+echo "== background =="
+# Git Bash mktemp → /tmp/…; native Deno writes a different path than bash polls.
+if [[ "${OSTYPE:-}" == msys* || "${OSTYPE:-}" == cygwin* ]]; then
+	marker_file="${ROOT}/.path-ci-hook-marker-$$"
+	: >"$marker_file"
+	if command -v cygpath >/dev/null 2>&1; then
+		export FOUNT_CI_HOOK_MARKER_FILE="$(cygpath -m "$marker_file")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)